include/ring-core/base.h

```diff
@@ -89,6 +89,9 @@
 #elif defined(__MIPSEL__) && defined(__LP64__)
 #define OPENSSL_64_BIT
 #define OPENSSL_MIPS64
+#elif defined(__riscv) && __riscv_xlen == 64
```

Comment:
BoringSSL uses `__SIZEOF_POINTER__`. I don't think that using the pointer size is as relevant for *ring* as it is for BoringSSL so I am open to using `__riscv_xlen` instead. Any thoughts on this?
```
#elif defined(__riscv) && __SIZEOF_POINTER__ == 8
#define OPENSSL_64_BIT
#define OPENSSL_RISCV64
```